src/status.cpp

```diff
@@ -191,12 +191,16 @@ const std::unordered_map<const StatusCode, const std::string> Status::statusMess
     {StatusCode::PIPELINE_INPUTS_AMBIGUOUS_METADATA, "Multiple nodes connected to the same pipeline input require different tensor metadata"},
 
     // Mediapipe
-    {StatusCode::MEDIAPIPE_GRAPH_START_ERROR, "Failed to start mediapipe graph"},
+    {StatusCode::MEDIAPIPE_GRAPH_START_ERROR, "Failed to start mediapipe graph: "},
```

Comment:
Not all MEDIAPIPE_GRAPH_START_ERROR are enforced to get details. So far the strategy was to add `: ` or `- ` as part of details.